### DISCUSSION TITLE
Settings to get back Karras variant?

### DISCUSSION THREAD

**Discussion by djwashout** [vladmandic/sdnext#2388] [General]
Just did a fresh install hash=53d0cb35 and I notice I don't have Karrass variant on samplers. Any way to get them back?

Using google colab notebook. 
updated=2023-10-19 hash=53d0cb35

**Comment by brknsoul** [comment]
It's a radio button under Sampler.
![image](https://github.com/vladmandic/automatic/assets/1969381/02f65860-bf63-4335-8b3e-ae28568645e5)

**Reply by djwashout** [reply]
Ohh wow. Thanks for pointing that out. Sorry for jumping to conclusions.

**Comment by djwashout** [comment]
Where am I going wrong here? I've made the selection, but images are coming out in DPM++ SDE 
![Screenshot 2023-10-22 181018](https://github.com/vladmandic/automatic/assets/132242585/d096271a-e3e4-4e48-ac69-ec10a095e055)
![Screenshot 2023-10-22 181110](https://github.com/vladmandic/automatic/assets/132242585/ea4fcace-7de1-46de-a33d-565941216a76)

**Reply by vladmandic** [reply]
what is the issue? i don't see a problem. NAME of the sampler is not the same as VARIANT used. and VARIANT is not used as namegen template.
full details are actually stored in metadata.

**Reply by djwashout** [reply]
I see now. Sorry form my oversight. I have looked in the medadata and I see the sigma listed as karras. It will take some time for me to adjust to this. Thanks for your help. 

![Screenshot 2023-10-22 223023](https://github.com/vladmandic/automatic/assets/132242585/f77da6f5-aaf2-4483-b427-83fd8e0436e5)

**Reply by brknsoul** [reply]
> It will take some time for me to adjust to this

You can simply set up your interface the way you want it to look on every start up, then go to System > Interface Defaults, and record them there.

**Reply by djwashout** [reply]
How do you set up the interface? Other than the prompt windows i don't see an option to drag or drop anything on the UI. i feel like i need a tutorial. lol.

**Reply by brknsoul** [reply]
I meant you change all your settings (sampler, height, width, extension settings, neg/pos prompts, etc) to how you'd like them to appear on start up, then you go to System > User Interface, then click "Set new defaults".
